src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -166,6 +166,414 @@ E. Click the *Tasks* tab, select an *Preview_Results* task and click on the *Pre
+Example:
+
+```python
+#%task(name=myTask,path=PATH_TO/IMPLEMENTATION_FILE.py)
+```
+
+===== Imports libraries
+
+The main difference between Proactive kernel and a native language one resides in the memory access during the
+execution of the different blocs.  While in a common native language kernel the whole script code (all the notebook
+blocs) is executed locally in the same shared memory zone, the ProActive kernel each created task will be executed as
+an independent process.
+To facilitate the transition from native language kernels to the ProActive one we included the pragma `#%import()`.
+This pragma gives the ability to the user to add, just once in the notebook, libraries that are common to all created
+tasks implemented in a same native script language.
+
+The pragma is used in this general manner: `#%import([language=SCRIPT_LANGUAGE])`. Example:
+
+```python
+#%import(language=Python)
+import os
+import pandas
+```
+
+Notice that if the language is not specified, Python is considered by default.
+
+===== Adding a fork environment
+
+To configure a fork environment for a task, please use the `#%fork_env()` pragma. A first way to do this
+is by providing the name of the corresponding task, and the fork environment implementation after that:
+
+```text
+#%fork_env(name=TASK_NAME)
+containerName = 'activeeon/dlm3'
+dockerRunCommand =  'docker run '
+dockerParameters = '--rm '
+paHomeHost = variables.get("PA_SCHEDULER_HOME")
+paHomeContainer = variables.get("PA_SCHEDULER_HOME")
+proActiveHomeVolume = '-v '+paHomeHost +':'+paHomeContainer+' '
+workspaceHost = localspace
+workspaceContainer = localspace
+workspaceVolume = '-v '+localspace +':'+localspace+' '
+containerWorkingDirectory = '-w '+workspaceContainer+' '
+preJavaHomeCmd = dockerRunCommand + dockerParameters + proActiveHomeVolume + workspaceVolume + containerWorkingDirectory + containerName
+```
+
+A second way is by providing the name of the task, and the path of a _.py_ file containing the fork environment code:
+
+```python
+#%fork_env(name=TASK_NAME, path=PATH_TO/FORK_ENV_FILE.py)
+```
+
+===== Adding a selection script
+
+To add a selection script to a task, please use the `#%selection_script()` pragma. A first way to do it,
+provide the name of the corresponding task, and the selection code implementation after that:
+
+```python
+#%selection_script(name=TASK_NAME)
+selected = True
+```
+
+A second way is by providing the name of the task, and the path of a .py file containing the selection code:
+
+```python
+#%selection_script(name=TASK_NAME, path=PATH_TO/SELECTION_CODE_FILE.py)
+```
+
+===== Adding job fork environment and/or selection script
+
+If the selection scripts and/or the fork environments are the same for all job tasks, we can add them just once using
+the `job_selection_script` and/or the `job_fork_env` pragmas.
+
+Usage:
+
+For a job selection script please use:
```

Comment:
Sometimes you use a comma before "please use", sometimes you don't, whichever you use just be consistent.
IMHO there are too many "please use", maybe we can remove them? the user knows we are being nice and polite ;)

Reply:
I just added the so many please after Amine's reviews on the other repo.
I will delete some.